bindings/python/crocoddyl/__init__.py

```diff
@@ -310,13 +310,13 @@ def _setConeMu(self, coneName, mu):
             self.frictionMu[coneName] = mu
             coneGroup = self.frictionGroup + "/" + coneName
 
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/0", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/1", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/2", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/3", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/cone", "")
-            self.robot.viewer.gui.deleteNode(coneGroup, "")
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/0", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/1", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/2", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/3", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/cone", True)
+            self.robot.viewer.gui.deleteNode(coneGroup, True)
```

Comment:
The boolean value toggle resistivity. As node *coneGroup* seems to be a group, calling `deleteNode(coneGroup, True)` automatically deletes child nodes so the last line would be sufficient here.

Reply:
What should I do in the previous lines? should I add `False` or do not include any argument?

Reply:
You can remove everything but the last line. `deleteNode(coneGroup, True)` will delete all nodes inside the group too.

This is not mandatory. You may prefer to keep the explicit deletion.

Reply:
I checked the animation in Gepetto, it is same with these codes and without theses codes. For clarity, I just deleted the first six lines in the new pr #898, but I am not sure which one is better.

Reply:
I will mark as resolved this discussion since the reviews are done in the PR.
Thanks guys :)